src/Components/ProfileDetails.jsx

```diff
@@ -0,0 +1,67 @@
+import { Text, VStack, Button, HStack, Icon } from "@chakra-ui/react";
+import { globe, moneyDollar, mail, linkedIn, twitter } from "../icons/icons";
+
+function ProfileDetails() {
+  return (
+    <>
+      <HStack padding="1rem" spacing="2rem">
+        <img
+          src="https://sm.mashable.com/t/mashable_in/photo/default/shark-tank-indias-ashneer-grover-might-get-fired-from-bharat_femk.h960.png"
+          alt="ashneer"
+          style={{
+            height: "150px",
+            width: "150px",
+            objectFit: "cover",
+            borderRadius: "50%",
+          }}
+        />
+        <VStack width={"80vw"}>
+          <HStack width="100%" spacing={6}>
+            <Text fontSize="3xl" fontWeight={650}>
```

Comment:
Please dont hard-code these values. Create a variable and then use it. Will be easier to integrate and show the data from the backend

Reply:
I'll discuss with rest of dudes, we'll go with variables